File: iOS/Trash Trek Sketch Planner/Trash Trek Sketch Planner/ViewController.m
//
//  ViewController.m
//  Trash Trek Scorer
//
//  Created by Sanjay Seshan on 8/31/15.
//  Copyright (c) 2015 EV3Lessons.com. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

// An example viewDidLoad
- (void)viewDidLoad
{
    [super viewDidLoad];
    //   self.webView.scalesPageToFit = YES;
    //  _webView.frame = CGRectMake(0.0, 0.0, 200.0, 700.0);
    [self loadWebView];
    
}

// The example loader
//
// Assumes you have an IBOutlet for the UIWebView defined:  @property (strong, nonatomic) UIWebView *wv;
- (void)loadWebView {
    // Remember that bundle resources do *not* have directories so all filenames must be unique.
    NSBundle *mainBundle = [NSBundle mainBundle];
    NSURL *homeIndexUrl = [mainBundle URLForResource:@"www/index" withExtension:@"html"];
    _webView = [UIWebView new];
    _webView.frame = self.view.bounds;
    [self.view addSubview:self.webView];
    // The magic is loading a request, *not* using loadHTMLString:baseURL:
    NSURLRequest *urlReq = [NSURLRequest requestWithURL:homeIndexUrl];
    self.webView.mediaPlaybackRequiresUserAction = NO;
    [self.webView loadRequest:urlReq];
    
}

@end

File: 2015-16/iOS/junk/Wheel Converter/Wheel Converter/ViewController.swift
//
//  ViewController.swift
//  TrashTrek Scorer
//
//  Created by Sanjay Seshan on 8/26/15.
//  Copyright (c) 2015 EV3Lessons.com. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    @IBOutlet var webView: UIWebView!
    override func viewDidLoad() {
        
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        //     let url = NSURL (string: "http://www.sourcefreeze.com");
        //    let requestObj = NSURLRequest(URL: url!);
        //    webView.loadRequest(requestObj);
        let localfilePath = NSBundle.mainBundle().URLForResource("www/index", withExtension: "html");
        let myRequest = NSURLRequest(URL: localfilePath!);
        webView.loadRequest(myRequest);
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    override func viewDidLayoutSubviews() {
        webView.frame = CGRectMake(0, 0, self.view.frame.size.width, self.view.frame.size.height);
    }
}

